src/Bicep.Core/Semantics/Namespaces/ThirdPartyNamespaceType.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using System.Collections.Immutable;
+using Bicep.Core.TypeSystem;
+using Bicep.Core.TypeSystem.Providers;
+using Bicep.Core.TypeSystem.Providers.ThirdParty;
+using Bicep.Core.TypeSystem.Types;
+
+namespace Bicep.Core.Semantics.Namespaces
+{
+    public static class ThirdPartyNamespaceType
+    {
+        public static NamespaceSettings Settings { get; } = new(
+            IsSingleton: true,
```

Comment:
>IsSingleton [](http://example.com/codeflow?start=12&length=11)

Out of scope of this PR, but eventually we will need to revisit what "Singleton" means here. With registry-based providers, I think the meaning will shift to one per provider name rather than just one.